website/blog/2025-02-26-tracing-introduction/index.mdx

```diff
@@ -0,0 +1,273 @@
+        {
+            "role": "user",
+            "content": "What is an MLflow tracking server?"
+        }
+    ]
+)
+```
+
+### Viewing your LLM Traces
+
+The MLflow UI provides an AI observability dashboard for viewing your traces. Start the MLflow UI from your terminal with:
+
+```shell
+mlflow ui
+```
+
+Navigate to the UI. the output of the `mlflow ui` command will tell you where to go (`http://localhost:5000` by default). In the UI, navigate to the "Traces" tab. This will list all of the collected traces. Click on a trace's Trace ID to open up a new pane with more details.
+
+![Traces in the MLflow UI](/img/blog/tracing-intro/01_tracing_ui.gif)
+
+:::tip
+
+By default, the MLflow server will listen on `http://localhost:5000`. You can choose a different port with `mlflow ui -p <port>`. For example, to listen on port 5001, use `mlflow ui -p 5001`.
+
+:::
+
+Starting the MLflow tracking server with `mlflow ui` also enables you to [view traces right in a Jupyter notebook](https://mlflow.org/blog/mlflow-tracing-in-jupyter)! You just have to set the tracking uri to the location specified above:
+
+```python
+mlflow.set_tracking_uri("http://localhost:5000")
+```
+
+Then, when you invoke an AI model/framework with tracing enabled, the generated trace(s) will appear right in the notebook outputs.
+
+![Tracing in Jupyter Notebooks](/img/blog/tracing-intro/02_jupyter.gif)
+
+You can disable this functionality with `mlflow.tracing.disable_notebook_display()`.
+
+
+## Organizing your Traces
+
+If you use tracing across multiple different projects and tasks, you might want to organize the traces into separate groups.
+
+The easiest way to organize your traces is to separate them into [*experiments*](https://mlflow.org/docs/latest/tracking.html#experiments). Each experiment has its own traces tab, which displays the traces for that experiment.
+
+You can create an experiment in the UI (With the "+" button next to "Experiments"), with the MLflow CLI, or with Python. Let's create a new experiment called "Quickstart" and log a trace.
+
+```python
+mlflow.set_experiment("quickstart")
+
+completion = client.chat.completions.create(
+    model="gpt-4o-mini",
+    messages=[
+        {"role": "system", "content": "You are a helpful assistant."},
+        {
+            "role": "user",
+            "content": "What is an MLflow tracking server?"
+        }
+    ]
+)
+```
+
+We can now find this trace in the "Traces" tab in the "quickstart" experiment.
+
+![Trace in Experiment](/img/blog/tracing-intro/03_experiment.png)
+
+The `set_experiment` function specifies which experiment traces should be logged to, creating it if it does not exist, so the code snippet above created a new "quickstart" experiment.
+
+You can also organize your traces with [tags](https://mlflow.org/docs/latest/llms/tracing/index.html#trace-tags) and [runs](https://mlflow.org/docs/latest/llms/tracing/index.html#q-how-can-i-associate-a-trace-with-an-mlflow-run).
```

Comment:
```suggestion
You can also organize your traces with [tags](https://mlflow.org/docs/latest/tracing#trace-tags) and [runs](https://mlflow.org/docs/latest/tracing#q-how-can-i-associate-a-trace-with-an-mlflow-run).
```
(updates for the new urls - the old ones have redirects, but we should use the new ones)